packages/astro/src/core/build/generate.ts

```diff
@@ -342,7 +283,7 @@ async function getPathsForRoute(
 		const label = staticPaths.length === 1 ? 'page' : 'pages';
 		logger.debug(
 			'build',
-			`├── ${bold(green('✔'))} ${route.component} → ${magenta(`[${staticPaths.length} ${label}]`)}`
+			`├── ${bold(green('√'))} ${route.component} → ${magenta(`[${staticPaths.length} ${label}]`)}`
```

Comment:
Woah, any reason to change to a square root symbol? 😅

Reply:
Not intended! Good catch